src/components/ec-input-field/ec-input-field.vue

```diff
@@ -11,9 +11,19 @@
     >
       <span
         v-if="label"
-        class="ec-input-field__label-text"
+        :class="{'ec-input-field__label-text': true, 'ec-input-field__label-text--tooltip': tooltipLabel}"
```

Comment:
We should use `--is-`, `--can-`, '--has-' prefixes for modifiers, e.g. `--has-tooltip`.